### DISCUSSION TITLE
Regular expression example on homepage is invalid

### DISCUSSION THREAD

**Discussion by Rylan12** [community/community#9959] [Other Feature Feedback, Questions, & Ideas] [answered]
On the code search homepage, the first example under the "Regular Expressions" category is [`/<Route(|>/`](https://cs.github.com/?q=%2F%3CRoute\(%08%7C%3E%2F)

<img width="200" alt="Screen Shot 2022-01-10 at 4 34 55 PM" src="https://user-images.githubusercontent.com/32525609/148843560-116b334a-ae1a-422a-a321-d410f7e5b918.png">
<img width="311" alt="Screen Shot 2022-01-10 at 4 40 10 PM" src="https://user-images.githubusercontent.com/32525609/148843584-0a5da235-1098-4dc0-bacf-39fecc6abc17.png">

Clicking this example gives an invalid regular expression error. At first I thought this was because the `(` wasn't escaped but there also seems to be some sort of invisible character between the `(` and the `|` (hence the weird formatting in the code block above) which makes me wonder if there's a bigger mistake.

I'm not totally sure what this regular expression was supposed to be but I'd guess that either something is supposed to  be escaped or a closing `)` is missing... Regardless of what was intended I think the link and example on the homepage shouldn't be invalid.

**Comment by colinwm** [comment, marked as answer]
This is fixed now! Thanks for reporting!
